snap/snapcraft.yaml

```diff
@@ -18,6 +18,9 @@ architectures:
   - build-on: [amd64]
     build-for: [amd64]
 
+system-usernames:
```

Comment:
It can be anything and we thought at first that the snap would change just based on the ubuntu version. I think that appending a number is a good idea, yes